src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -1632,13 +1632,19 @@ public static IEnumerable<object[]> VerifyUploadServersStreamsAndExpectedData
 
         #region Various HTTP Method Tests
 
-        [ActiveIssue(22161, TargetFrameworkMonikers.Uap)]
         [OuterLoop] // TODO: Issue #11345
         [Theory, MemberData(nameof(HttpMethods))]
         public async Task SendAsync_SendRequestUsingMethodToEchoServerWithNoContent_MethodCorrectlySent(
             string method,
             bool secureServer)
         {
+            if (PlatformDetection.IsUap && method == "TRACE")
+            {
+                // 'TRACE' method is not supported on Uap.
+                // See: https://github.com/dotnet/corefx/issues/22161
+                return;
```

Comment:
It hard to map that particular HttpRequestException into something specific for 'TRACE'.

Based on this feedback, I will change the code to throw PNSE directly with a useful message and adjust the tests accordingly.